web/src/app/selection/QuerySelect.js

```diff
@@ -55,13 +56,11 @@ function makeUseValues(query, mapNode) {
     value.forEach((v, i) => {
       variables['id' + i] = v
     })
-
-    const { data, error } = useQuery(getQueryBySize(value.length), {
-      skip: !value.length,
+    const [{ data, error }] = useQuery({
+      query: print(getQueryBySize(value.length)),
+      pause: !value.length,
       variables,
-      returnPartialData: true,
-      fetchPolicy: 'cache-first',
-      pollInterval: 0,
```

Comment:
same here

Reply:
it's enough to just set `cache-first` with the new code. default is `cache-and-network` (which will automatically poll)